CustomKeystore/src/main/java/com/amazonaws/services/kinesisanalytics/CustomFlinkKafkaUtil.java

```diff
@@ -0,0 +1,34 @@
+package com.amazonaws.services.kinesisanalytics;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class CustomFlinkKafkaUtil {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CustomFlinkKafkaUtil.class);
+
+    public static void initializeKafkaTrustore() {
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        File destDir = new File("/tmp");
+        File targetFile = new File(destDir, "kafka.client.truststore.jks");
+
+        try (InputStream input = classLoader.getResourceAsStream("kafka.client.truststore.jks");
+             OutputStream outStream = new FileOutputStream(targetFile)) {
+
+            int available = input.available();
+            byte[] buffer = new byte[available];
+            input.read(buffer);
+
+            outStream.write(buffer);
+            outStream.flush();
+        } catch (Exception ex) {
+            LOG.error(ex.getMessage());
```

Comment:
@jeremyber-aws copied this from the previous implementation. But I would rather fail the job here with a sensible error message. What do you think?